settings.py

```diff
@@ -220,15 +215,17 @@
     DEBUG = False
 else:
     DEBUG = True
-    # Enable CORS for local development
-    INSTALLED_APPS.append('corsheaders')
+    # Enable CORS and hijak for local development
+    INSTALLED_APPS = ['corsheaders', 'hijack'] + INSTALLED_APPS
```

Comment:
It reports `Forbidden (CSRF token missing or incorrect.)` in my environment. Are you sure it works in your environment?

Reply:
good catch, I messed up a couple settings. I'll push up the change

Reply:
this is fixed